test/unit/sessions/softphone-session-handler.test.ts

```diff
@@ -267,9 +267,9 @@ describe('acceptSession()', () => {
     const defaultElement = {};
     const defaultStream = new MockStream({ audio: true });
 
-    mockSdk._config.defaults.audioElement = defaultElement as any;
-    mockSdk._config.defaults.audioStream = defaultStream as any;
-    const volume = mockSdk._config.defaults.audioVolume = 67;
+    mockSdk._config.defaults!.audioElement = defaultElement as any;
+    mockSdk._config.defaults!.audioStream = defaultStream as any;
```

Comment:
Codacy has a fix for the issue: [Forbidden non-null assertion.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9637746)
```suggestion
    mockSdk._config.defaults?.audioStream = defaultStream as any;
```